modules/OCPP/OCPP.cpp

```diff
@@ -32,6 +32,14 @@ static ocpp1_6::ChargePointErrorCode get_ocpp_error_code(const Object& evse_erro
     }
 }
 
+void create_user_config(const boost::filesystem::path& user_config_path) {
```

Comment:
should be rather called create_empty_user_config, because it creates an empty config